packages/utils/src/types/primitives.ts

```diff
@@ -103,6 +103,10 @@ export class Address {
   sortPair(b: Address): [Address, Address] {
     return this.lt(b) ? [this, b] : [b, this]
   }
+
+  static createMock(): Address {
+    return Address.fromString('0xCf7Ed3AccA5a467e9e704C703E8D87F634fB0Fc9')
```

Comment:
We can probably have this as a constant but not crucial. What does it return? Might be worth adding what it returns as a comment for when people `grep` or search for this.

Reply:
it's really just a shortcut to make it easier to write tests. it returns a valid address.